Adding PrintingController (#85)
Grupo-AFAL/frontend-helpers: javascript/docs/print.md

---
title: Print
description: This controller allows to print the current page when Print Controller connect
---

### Print Dialog from Anchor (HTML A tag)

> Click print and Print Dialog will comes in.

Reviewer: ```suggestion
Click on Print and a Print Dialog will appear.
```
